gadget/update.go

```diff
@@ -244,13 +246,15 @@ func isCompatibleSchema(gadgetSchema, diskSchema string) bool {
 		return diskSchema == "gpt"
 	case "mbr":
 		return diskSchema == "dos"
-	case "emmc":
-		return diskSchema == "emmc"
```

Comment:
I think it is fine to leave this here, we might want to use this function in other cases and forget about adding the emmc case